admin/src/templates/index.html

```diff
@@ -12,17 +12,17 @@
       <div class="body container">
         <div class="navigation">
           <ul>
-            <li ng-class="{'active-nav': checkActive('settings')}"><a ui-sref="settings.basic" translate>Settings</a></li>
-            <li ng-class="{'active-nav': checkActive('translation')}"><a ui-sref="translation.languages" translate>Languages</a></li>
-            <li ng-class="{'active-nav': checkActive('forms')}"><a ui-sref="forms.xml" translate>Forms</a></li>
+            <li ng-class="{'active-nav': checkActive('settings')}" mm-auth="can_configure"><a ui-sref="settings.basic" translate>Settings</a></li>
+            <li ng-class="{'active-nav': checkActive('translation')}" mm-auth="can_configure"><a ui-sref="translation.languages" translate>Languages</a></li>
+            <li ng-class="{'active-nav': checkActive('forms')}" mm-auth="can_configure"><a ui-sref="forms.xml" translate>Forms</a></li>
             <li ng-class="{'active-nav': checkActive('export')}"><a ui-sref="export.messages" translate>import.export</a></li>
-            <li ui-sref-active="active-nav"><a ui-sref="upgrade" translate>instance.upgrade</a></li>
-            <li ui-sref-active="active-nav"><a ui-sref="users" translate>Users</a></li>
-            <li ng-class="{'active-nav': checkActive('images')}"><a ui-sref="images.icons" translate>images</a></li>
-            <li ng-class="{'active-nav': checkActive('targets')}"><a ui-sref="targets" translate>analytics.targets</a></li>
-            <li ng-class="{'active-nav': checkActive('authorization')}"><a ui-sref="authorization.permissions" translate>configuration.authorization</a></li>
-            <li ui-sref-active="active-nav"><a ui-sref="messagetest" translate>configuration.messagetest</a></li>
-            <li ng-class="{'active-nav': checkActive('message-queue')}"><a ui-sref="message-queue.due" translate>admin.message.queue</a></li>
+            <li ui-sref-active="active-nav" mm-auth="can_configure"><a ui-sref="upgrade" translate>instance.upgrade</a></li>
+            <li ui-sref-active="active-nav" mm-auth="can_configure"><a ui-sref="users" translate>Users</a></li>
+            <li ng-class="{'active-nav': checkActive('images')}" mm-auth="can_configure"><a ui-sref="images.icons" translate>images</a></li>
+            <li ng-class="{'active-nav': checkActive('targets')}" mm-auth="can_configure"><a ui-sref="targets" translate>analytics.targets</a></li>
+            <li ng-class="{'active-nav': checkActive('authorization')}" mm-auth="can_configure"><a ui-sref="authorization.permissions" translate>configuration.authorization</a></li>
+            <li ui-sref-active="active-nav" mm-auth="can_configure"><a ui-sref="messagetest" translate>configuration.messagetest</a></li>
+            <li ng-class="{'active-nav': checkActive('message-queue')}" mm-auth mm-auth-any="[ 'can_configure', 'can_view_outgoing_messages' ]"><a ui-sref="message-queue.due" translate>admin.message.queue</a></li>
```

Comment:
This tab has nothing to do with configuration so we shouldn't check for the `can_configure` permission - use `can_view_outgoing_messages` only.